products/batch_exports/backend/temporal/destinations/utils.py

```diff
@@ -0,0 +1,21 @@
+import datetime as dt
+
+
+def _get_query_timeout(data_interval_start: dt.datetime | None, data_interval_end: dt.datetime) -> float:
+    """Get the timeout to use for long running queries.
+
+    Operations like COPY INTO TABLE and MERGE can take a long time to complete, especially if there is a lot of data and
+    the instance being used is not very powerful. We don't want to allow these queries to run for too long, as they can
+    cause SLA violations and can consume a lot of resources in the user's instance.
+    """
```

Comment:
Same function that used to be in `snowflake_batch_export.py` but I slightly updated the description to be more generic (warehouse -> instance, compute resources -> resources).